.github/workflows/snyk.yml

```diff
@@ -0,0 +1,18 @@
+name: Snyk
+on:
+  schedule:
```

Comment:
running on push to main is enough, you don't need the daily schedule, as your dependencies won't change between pushes

Reply:
I wonder when this anti-pattern first appeared. We should remove all occurrences of it to avoid future copy+pasting (https://github.com/search?q=org%3Aguardian+path%3Asnyk.yml+schedule&type=code).

Is there an example `snyk.yml` that we should be copying from rather than just picking a random repo?